packages/app/components/Nav/NavBar.tsx

```diff
@@ -180,13 +180,16 @@ export default function NavBar({ courseId }: NavBarProps): ReactElement {
       text: "Queue",
     });
   }
+
+  /**
```

Comment:
Nit: just add a TODO here so we know to re-enable it eventually (maybe link this issue on GH)

Reply:
good idea!